src/main/java/aquality/selenium/core/utilities/IUtilitiesModule.java

```diff
@@ -5,6 +5,13 @@
  */
 public interface IUtilitiesModule {
 
+    /**
+     * @return implementation of {@link IActionRetrier}.
+     */
+    default Class<? extends IActionRetrier> getActionRetrierImplementation() {
```

Comment:
Maybe remove suffix `Implementation`?

Reply:
We discussed this naming locally when implement Modules and decided to use this form, because ActionRetrier but its implementation